bin/rad-issues

```diff
@@ -26,19 +26,38 @@
       [(string-append (show key) " - " (lookup :title val)) val]))
     (def iss (dict-from-seq (map mk-key (seq (list-issues)))))
     (def selected (fzf-select-by-key! iss))
+    (def comment-divider "---")
+    (def username
+      (fn [d]
+        (catch 'any (lookup :github-username d) (fn [x] "???"))))
+    (def issue-number (fn [d] (lookup :number d)))
+    (def body (fn [d] (lookup :body d)))
     (def pretty-comment (fn [c]
       (string-append
-        "* **" (lookup :github-username c) "** [" (lookup :created-at c) "]:\n\n"
-        (string/unlines (map (fn [xs] (string-append "   " xs)) (string/lines (lookup :body c)))))))
+        "* **" (username c) "** [" (lookup :created-at c) "]:\n\n"
+        (string/intercalate "\n\n" (map (fn [xs] (string-append "   " xs)) (string/lines (body c)))))))
```

Comment:
Why the empty lines? I.e., why not use `string/unlines`?

Reply:
`string/unlines` adds `\n` between the lines, this was not detected as a linebreak by the markdown preview though.

Reply:
Ah I see.